portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Jennifer's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <header>
+    <h1>JENNIFER JIMENEZ</h1>
+    <div class="logos">
+      <a href="https://www.linkedin.com/in/jen-jimenez" target="_blank"> 
+        <img src="/images/linkedin.png" id="logo"/>
+      </a>
+      <a href="https://github.com/jennifer-jimenez" target="_blank"> 
+        <img src="/images/git.png" id="logo"/>
+      </a>
+      <a href="https://www.instagram.com/prettyoddjen/" target="_blank"> 
+        <img src="/images/ig.png" id="logo"/>
```

Comment:
See my comment about `id`s on index.html. I think most of the comments on that file apply here too but I won't repeat them all.